.mergify.yml

```diff
@@ -20,6 +20,16 @@ pull_request_rules:
     actions:
       queue:
 
+  - name: backport 5.10
+    actions:
+      backport:
+        ignore_conflicts: true
+        branches:
+          - '5.10'
+    conditions:
+      - label=ready-to-merge
+      - label=v/5.10
+
```

Comment:
just a reminder about the label, I saw 6.0 in https://github.com/crate/crate/labels?page=5&sort=name-asc but not 5.10